app/controllers/posts_controller.rb

```diff
@@ -58,7 +58,12 @@ def destroy
 
   # GET /about
   def about
-    # @post = Post.find(???)
+    begin
+      @post = Post.find_by! static_id: :about
+    rescue ActiveRecord::RecordNotFound
+      @post = Post.create static_id: :about, title: 'Rólunk oldal tartalma', short_description: '[Nem szükséges kitölteni]'
```

Comment:
We should use `create! ` to raise exceptions when saving a record faild.
It could make finding bugs easier in the future.

Reply:
I guess there should be a separate issue for this one: to change all of these `create`, `update`, `destroy` methods used on the `ActiveRecord`s/Models. :thinking:

Reply:
I don't think we should change it in all place.  The scaffolded code uses if - else statement to handle errors. where the `!
` less methods are used, beacause they return a boolean, wich indicates if the operation is successful or not. If we use it this I think it is useful.

I prefer to use the ActiveRecord methods with `!` when we don't want to explicitly handle unsucessful operations, but we want to know about them.